spread.yaml

```diff
@@ -75,6 +75,9 @@ backends:
             - fedora-27-64:
                 image: fedora-27-64-selinux-permissive
                 workers: 4
+            - fedora-28-64:
```

Comment:
In your comment you said that fedora 28 would remain manual but it is not manual here. Do you plan to switch it to manual after a round of tests that show this can be merged as working?